conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -33,10 +33,10 @@ POST        /manage-transit-movements/test-only/departure/message
 POST        /manage-transit-movements/test-only/departure-outbound                                                      controllers.testOnly.TestOnlyP5DeparturesAPIController.outboundDepartureMessage()
 
 + nocsrf
-POST        /manage-transit-movements/test-only/departure-inbound/:departureId                                          controllers.testOnly.TestOnlyP5DeparturesAPIController.inboundDepartureMessage(departureId: String)
+POST        /manage-transit-movements/test-only/:departureId/departure-inbound                                          controllers.testOnly.TestOnlyP5DeparturesAPIController.inboundDepartureMessage(departureId: String)
 
 + nocsrf
-POST        /manage-transit-movements/test-only/departure-outbound-message/:departureId                                 controllers.testOnly.TestOnlyP5DeparturesAPIController.addMessageToDeparture(departureId: String)
+POST        /manage-transit-movements/test-only/:departureId/departure-outbound-message                               controllers.testOnly.TestOnlyP5DeparturesAPIController.addMessageToDeparture(departureId: String)
 
 + nocsrf
 GET         /manage-transit-movements/test-only/departure/:departureId/message/:messageId                               controllers.testOnly.TestOnlyP5DeparturesAPIController.departureMessage(departureId: String, messageId: String)
```

Comment:
for consistency would you not want :
manage-transit-movements/test-only/:departureId/departure/message/:messageId
controllers.testOnly.TestOnlyP5DeparturesAPIController.departureMessage(departureId: String, messageId: String)